### ISSUE TITLE
Layout, Assignment tab gone after upgrade to gantry 5.1.3

### ISSUE THREAD

**Issue by axel-rank** [gantry/gantry5#842]
If I navigate to gantry config with
Joomla Control Panel ->Gantry 5 themes -->Available themes -->Configure
Layout and Assignment tabs are missing
![screenshot](https://cloud.githubusercontent.com/assets/12480617/9897426/8ce322c8-5c48-11e5-8abd-3802cd3b08a1.png)
If I navigate to gantry config with
Extensions -->Templates-->Hydrogen - Standard
Everything is ok
![screenshot2](https://cloud.githubusercontent.com/assets/12480617/9897448/e2d362a6-5c48-11e5-8d7b-a089ed047314.png)

**Comment by w00fz** [comment]
It's not present only for the Base Outline and has been like this in Joomla for many versions now.

In Joomla the Base Outline is only used for setting default values for Styles and Settings. You alsp cannot assign the Base Outline to any menu item, that's why also the Assignments tab is not there.

This is not the case for WordPress where the Base Outline works like a generic outline, reason being that WordPress has different concepts over joomla.

Bottom line, this is not an issue, has been like that for at least the past 3-4 releases :)

// Djamil Legato // From iPhone //
